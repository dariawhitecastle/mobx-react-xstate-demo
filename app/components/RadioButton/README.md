<!-- ! This is a generated file. To make changes, edit <Component>.doc.js ! -->
## createRadioButton
A radio button control.

RadioButton should typically not be used directly.
      Instead, use RadioButtonGroup.

## Usage

```javascript
import { RadioButton } from 'components/RadioButton';
<RadioButton />
```

## Properties

**checked**

Same as React <input checked={} />

```
boolean
```

**disabled**

Same as React <input disabled={} />. Also adds a hidden input element
with the same name so form submissions work.

```
boolean
```

**id**

The DOM id attribute value to use for the underlying <input/> element.

```
string
```

**label**

Label text to place next to the control.

```
node
```

**name**

Required. The DOM name attribute value to use for the underlying <input/> element.

```
string
```

**onChange**

Function that will be called when the user clicks the radio button. It
      will be passed a React event object. The current state can be accessed
      via event.target.checked. Same as React <input onChange={} />.

```
function
```

## Intrinsic element

```
input
```
## Theme

**radioButton.border.color**

The color of the border of the Radio Button. Expects `string | { dark: string, light: string }`.

Defaults to

```
{dark: 'rgba(255, 255, 255, 0.5), light: 'rgba(0, 0, 0, 0.15)}
```

**radioButton.border.width**

The width size of the border of the RadioButton. Expects `string`.

Defaults to

```
2px
```

**radioButton.check.color**

The color of the checked icon in the RadioButton. Expects `string | { dark: string, light: string }`.

Defaults to

```
undefined
```

**radioButton.check.extend**

Any additional style for the checked RadioButton. Expects `string | (props) => {}`.

Defaults to

```
undefined
```

**radioButton.check.radius**

The border-radius of the RadioButton. Expects `string`.

Defaults to

```
100%
```

**radioButton.extend**

Any additional style for the RadioButton. Expects `string | (props) => {}`.

Defaults to

```
undefined
```

**radioButton.gap**

The gap between the label and the RadioButton itself. Expects `string`.

Defaults to

```
small
```

**radioButton.size**

The size of the RadioButton. Expects `string`.

Defaults to

```
24px
```

**radioButton.hover.border.color**

The color of the RadioButton border when hovered over. Expects `string | { dark: string, light: string }`.

Defaults to

```
{dark: white, light: black}
```

**radioButton.icon.extend**

Any additional style for the RadioButton icon. Expects `string | (props) => {}`.

Defaults to

```
undefined
```

**radioButton.icon.size**

The size of the icon in the RadioButton. Expects `string`.

Defaults to

```
undefined
```
