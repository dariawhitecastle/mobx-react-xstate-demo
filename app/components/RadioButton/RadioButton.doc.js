import { describe, PropTypes } from 'react-desc'

export const doc = RadioButton => {
  const DocumentedRadioButton = describe(RadioButton)
    .description('A radio button control.')
    .details(
      `RadioButton should typically not be used directly.
      Instead, use RadioButtonGroup.`,
    )
    .usage(
      `import { RadioButton } from 'components/RadioButton';
<RadioButton />`,
    )
    .intrinsicElement('input')

  DocumentedRadioButton.propTypes = {
    checked: PropTypes.bool.description('Same as React <input checked={} />'),
    disabled: PropTypes.bool.description(
      `Same as React <input disabled={} />. Also adds a hidden input element
with the same name so form submissions work.`,
    ),
    id: PropTypes.string.description(
      'The DOM id attribute value to use for the underlying <input/> element.',
    ),
    label: PropTypes.node.description('Label text to place next to the control.'),
    name: PropTypes.string.description(
      'The DOM name attribute value to use for the underlying <input/> element.',
    ).isRequired,
    onChange: PropTypes.func.description(
      `Function that will be called when the user clicks the radio button. It
      will be passed a React event object. The current state can be accessed
      via event.target.checked. Same as React <input onChange={} />.`,
    ),
  }

  return DocumentedRadioButton
}

export const themeDoc = {
  'radioButton.border.color': {
    description: 'The color of the border of the Radio Button.',
    type: 'string | { dark: string, light: string }',
    defaultValue: "{dark: 'rgba(255, 255, 255, 0.5), light: 'rgba(0, 0, 0, 0.15)}",
  },
  'radioButton.border.width': {
    description: 'The width size of the border of the RadioButton.',
    type: 'string',
    defaultValue: '2px',
  },
  'radioButton.check.color': {
    description: 'The color of the checked icon in the RadioButton.',
    type: 'string | { dark: string, light: string }',
    defaultValue: 'undefined',
  },
  'radioButton.check.extend': {
    description: 'Any additional style for the checked RadioButton.',
    type: 'string | (props) => {}',
  },
  'radioButton.check.radius': {
    description: 'The border-radius of the RadioButton.',
    type: 'string',
    defaultValue: '100%',
  },
  'radioButton.extend': {
    description: 'Any additional style for the RadioButton.',
    type: 'string | (props) => {}',
  },
  'radioButton.gap': {
    description: 'The gap between the label and the RadioButton itself.',
    type: 'string',
    defaultValue: 'small',
  },
  'radioButton.size': {
    description: 'The size of the RadioButton.',
    type: 'string',
    defaultValue: '24px',
  },
  'radioButton.hover.border.color': {
    description: `The color of the RadioButton border when hovered over.`,
    type: 'string | { dark: string, light: string }',
    defaultValue: '{dark: white, light: black}',
  },
  'radioButton.icon.extend': {
    description: 'Any additional style for the RadioButton icon.',
    type: 'string | (props) => {}',
  },
  'radioButton.icon.size': {
    description: 'The size of the icon in the RadioButton.',
    type: 'string',
  },
}
