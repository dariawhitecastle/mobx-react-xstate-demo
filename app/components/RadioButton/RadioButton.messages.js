/*
 * RadioButton Messages
 *
 * This contains all the text for the RadioButton component.
 */

import { defineMessages } from 'react-intl'

export const scope = 'app.components.RadioButton'

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the RadioButton component!',
  },
})
