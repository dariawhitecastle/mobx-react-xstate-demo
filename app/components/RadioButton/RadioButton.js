/* eslint-disable react/prop-types */
/**
 * RadioButton
 */

import React from 'react'
import { RadioButton } from 'grommet'

import { doc } from './RadioButton.doc'
import helpers from '../../utils/helpers'

const createRadioButton = ({ checked, disabled, id, label, name, onChange, ...rest }) => (
  <RadioButton
    checked={checked}
    disabled={disabled}
    id={id}
    label={label}
    name={name}
    onChange={onChange}
    {...rest}
  />
)

export default helpers.createWithDoc({
  docFunction: doc,
  component: createRadioButton,
})
