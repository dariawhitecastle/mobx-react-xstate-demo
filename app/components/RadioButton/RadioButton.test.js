import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import { Grommet } from 'grommet'
import RadioButton from './RadioButton'

test('RadioButton renders', () => {
  const component = renderer.create(
    <Grommet>
      <RadioButton name="test empty" />
      <RadioButton id="test id" name="test name" />
    </Grommet>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('RadioButton label renders', () => {
  const component = renderer.create(
    <Grommet>
      <RadioButton label="test label" name="test label" />
      <RadioButton label={<div>test label</div>} name="test div label" />
    </Grommet>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('RadioButton checked renders', () => {
  const component = renderer.create(
    <Grommet>
      <RadioButton checked name="test checked" />
    </Grommet>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('RadioButton disabled renders', () => {
  const component = renderer.create(
    <Grommet>
      <RadioButton disabled name="test disabled" />
      <RadioButton disabled checked name="test checked disabled" />
    </Grommet>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
