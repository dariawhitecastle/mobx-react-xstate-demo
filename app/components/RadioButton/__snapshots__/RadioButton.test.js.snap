// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton checked renders 1`] = `
.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-right: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  height: 24px;
  width: 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: solid 2px #7D4CDB;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0px;
  border-radius: 100%;
}

.c0 {
  font-size: 18px;
  line-height: 24px;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c1 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.c1:hover input:not([disabled]) + div,
.c1:hover input:not([disabled]) + span {
  border-color: #000000;
}

.c4 {
  opacity: 0;
  -moz-appearance: none;
  width: 0;
  height: 0;
  margin: 0;
  cursor: pointer;
}

.c6 {
  box-sizing: border-box;
  width: 24px;
  height: 24px;
  fill: #7D4CDB;
}

@media only screen and (max-width:768px) {
  .c2 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    border: solid 2px #7D4CDB;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    padding: 0px;
  }
}

<div
  className="c0"
>
  <label
    className="c1 c2"
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        checked={true}
        className="c4"
        name="test checked"
        type="radio"
      />
      <div
        className="c5"
      >
        <svg
          className="c6"
          preserveAspectRatio="xMidYMid meet"
          viewBox="0 0 24 24"
        >
          <circle
            cx={12}
            cy={12}
            r={6}
          />
        </svg>
      </div>
    </div>
  </label>
</div>
`;

exports[`RadioButton disabled renders 1`] = `
.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-right: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  height: 24px;
  width: 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: solid 2px rgba(0,0,0,0.15);
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0px;
  border-radius: 100%;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  height: 24px;
  width: 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: solid 2px #7D4CDB;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0px;
  border-radius: 100%;
}

.c0 {
  font-size: 18px;
  line-height: 24px;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c1 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  opacity: 0.5;
  cursor: default;
}

.c1:hover input:not([disabled]) + div,
.c1:hover input:not([disabled]) + span {
  border-color: #000000;
}

.c4 {
  opacity: 0;
  -moz-appearance: none;
  width: 0;
  height: 0;
  margin: 0;
}

.c7 {
  box-sizing: border-box;
  width: 24px;
  height: 24px;
  fill: #7D4CDB;
}

@media only screen and (max-width:768px) {
  .c2 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    border: solid 2px rgba(0,0,0,0.15);
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c6 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c6 {
    border: solid 2px #7D4CDB;
  }
}

@media only screen and (max-width:768px) {
  .c6 {
    padding: 0px;
  }
}

<div
  className="c0"
>
  <label
    className="c1 c2"
    disabled={true}
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        className="c4"
        disabled={true}
        name="test disabled"
        type="radio"
      />
      <div
        className="c5"
      />
    </div>
  </label>
  <label
    className="c1 c2"
    disabled={true}
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        checked={true}
        className="c4"
        disabled={true}
        name="test checked disabled"
        type="radio"
      />
      <div
        className="c6"
      >
        <svg
          className="c7"
          preserveAspectRatio="xMidYMid meet"
          viewBox="0 0 24 24"
        >
          <circle
            cx={12}
            cy={12}
            r={6}
          />
        </svg>
      </div>
    </div>
  </label>
</div>
`;

exports[`RadioButton label renders 1`] = `
.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-right: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  height: 24px;
  width: 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: solid 2px rgba(0,0,0,0.15);
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0px;
  border-radius: 100%;
}

.c0 {
  font-size: 18px;
  line-height: 24px;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c1 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.c1:hover input:not([disabled]) + div,
.c1:hover input:not([disabled]) + span {
  border-color: #000000;
}

.c4 {
  opacity: 0;
  -moz-appearance: none;
  width: 0;
  height: 0;
  margin: 0;
  cursor: pointer;
}

@media only screen and (max-width:768px) {
  .c2 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    border: solid 2px rgba(0,0,0,0.15);
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    padding: 0px;
  }
}

<div
  className="c0"
>
  <label
    className="c1 c2"
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        className="c4"
        name="test label"
        type="radio"
      />
      <div
        className="c5"
      />
    </div>
    <span>
      test label
    </span>
  </label>
  <label
    className="c1 c2"
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        className="c4"
        name="test div label"
        type="radio"
      />
      <div
        className="c5"
      />
    </div>
    <div>
      test label
    </div>
  </label>
</div>
`;

exports[`RadioButton renders 1`] = `
.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-right: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  height: 24px;
  width: 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: solid 2px rgba(0,0,0,0.15);
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0px;
  border-radius: 100%;
}

.c0 {
  font-size: 18px;
  line-height: 24px;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c1 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.c1:hover input:not([disabled]) + div,
.c1:hover input:not([disabled]) + span {
  border-color: #000000;
}

.c4 {
  opacity: 0;
  -moz-appearance: none;
  width: 0;
  height: 0;
  margin: 0;
  cursor: pointer;
}

@media only screen and (max-width:768px) {
  .c2 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    border: solid 2px rgba(0,0,0,0.15);
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    padding: 0px;
  }
}

<div
  className="c0"
>
  <label
    className="c1 c2"
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        className="c4"
        name="test empty"
        type="radio"
      />
      <div
        className="c5"
      />
    </div>
  </label>
  <label
    className="c1 c2"
    htmlFor="test id"
    onClick={[Function]}
  >
    <div
      className="c3"
    >
      <input
        className="c4"
        id="test id"
        name="test name"
        type="radio"
      />
      <div
        className="c5"
      />
    </div>
  </label>
</div>
`;
